snoop/models.py

```diff
@@ -46,7 +46,12 @@ class ArchiveListCache(models.Model):
     time = models.DateTimeField(auto_now=True)
 
 class Document(models.Model):
-    container = models.ForeignKey('Document', null=True)
+    container = models.ForeignKey('Document',
+                                  related_name='snoop_document_container',
```

Comment:
We can drop the `snoop_` prefix, this name is only used in Python code, not postgres.